libraries/BLE/src/BLERemoteCharacteristic.cpp

```diff
@@ -468,7 +475,7 @@ void BLERemoteCharacteristic::registerForNotify(notify_callback notifyCallback,
 		if(!notifications) val[0] = 0x02;
 		BLERemoteDescriptor* desc = getDescriptor(BLEUUID((uint16_t)0x2902));
 		if (desc != nullptr)
-			desc->writeValue(val, 2);
+			desc->writeValue(val, 2, true);
```

Comment:
What is the reason to response await here?

Reply:
I gave the details in the commit that introduces the change. For convenience, the commit message is:

> When registering a notification on a characteristic, the 2902 descriptor
> (CCCD) value is set to 1 (or 2 for indication).
> According to the BLUETOOTH CORE SPECIFICATION Version 5.2, Revision Date
> 2019-12-31, section 4.12.3 "Write Characteristic Descriptors" (page 1588),
> the characteristic descriptor write must expect a response.
> Currently, the descriptor write is performed without expecting a reponse,
> which prevents the notification to be functional with some BLE stacks.
> This commit modify the write to expect the response.

Reply:
Ok, thanks. 
Commit aproved.

Reply:
Thank you.